events.ts

```diff
@@ -42,27 +42,59 @@
             continue;
           }
 
-          let selectorElement;
           if (node.matches(selector)) {
             console.debug('<experiment> Selector matches', selector);
-            console.debug('<experiment> callback', node);
-
-            callback(node);
+            foundSelector(node);
           } else if ((selectorElement = node.querySelector(selector))) {
             console.debug('<experiment> Selector exist in node', selector);
-            observer.disconnect();
-
-            console.debug('<experiment> callback', selectorElement);
-            callback(selectorElement);
+            foundSelector(selectorElement);
           }
         }
       }
     });
 
-    console.debug('<experiment> observing', element);
-    observer.observe(element, {
+    console.debug('<experiment> observing', wrapper);
+    observer.observe(wrapper, {
       childList: true,
       subtree: true,
     });
-  }
+  },
+};
+
+(() => {
+  const addIdentifierClasses = (element, id) => {
+    element.classList.add('Experiment', id);
+  };
+
+  const eventToDynamicYield = () => {
+    DY.API('event', {
```

Comment:
Hello @coopjakob,

There is a change DY is not setup if someone didn't accept required consents or the DY is simply disabled. Needs a check for if DY object and API function exists.

Reply:
Thanks for the comment! I'm loading the `event.js` script from Dynamic Yield using a "Site personalization", so it `DY` should exist. You can find the setup in the "Stg" site. It's named "abtest=true".

Reply:
OK then it should be fine